app/index/template.hbs

```diff
@@ -13,19 +11,34 @@
 			<div class="row">
 				<div class="col-xs-12">
 					<div class="cta-container">
-							<div class="cta-text">
-								<h3> Not seeing a feed you want? Contribute! </h3>
-							</div>
-							<div class="cta-btn">
-								{{#link-to "feeds.new" class="btn btn-action" role="button"}}
-										Add a Feed
-								{{/link-to}}
-							</div>
+						<div class="cta-text">
+							<h3>Know a feed that's missing?</h3>
+						</div>
+						<div class="cta-btn">
+							{{#link-to "feeds.new" class="btn btn-action" role="button" }}
+								Add a Feed
+							{{/link-to}}
 						</div>
+					</div>
 				</div>
 			</div>
+			<p>
+				We would love your help to expand Transitland's coverage throughout the world. If you know of a feed, {{#link-to "feeds.new"}}add it here{{/link-to}}. To report an error in an existing entry, you can send a note to <a href="mailto:transitland@mapzen.com">transitland@mapzen.com</a>
+			</p>
+		{{else}}
```

Comment:
@drewda What does the if/else do here? 29-31 look like what's there publicly now?